db/version_edit.cc

```diff
@@ -244,6 +244,11 @@ bool VersionEdit::EncodeTo(std::string* dst) const {
       PutVarint64(&varint_tail_size, f.tail_size);
       PutLengthPrefixedSlice(dst, Slice(varint_tail_size));
     }
+    if (!f.user_defined_timestamps_persisted) {
+      PutVarint32(dst, NewFileCustomTag::kUserDefinedTimestampsPersisted);
+      char p = static_cast<char>(0);
```

Comment:
We could put a comment here explaining that the default is true and we're putting 0 as the value here to signal false (i.e. UDTS not persisted).

Reply:
Thanks for the suggestion for a comment here! It's indeed not self-explanatory why 0 has to be saved.